### ISSUE TITLE
Provide API to add jobs to existing experiment

### ISSUE THREAD

**Issue by patricksnape** [menpo/workerbee#3]
This will make it simpler to have long running jobs.

**Comment by jabooth** [comment]
Yup, this would be very useful. It would also simply the API which I think is currently a little confusing (that job_ids are ignored if the experiment already exists).
